### ISSUE TITLE
files_external 3rdparty composer errors

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#40445]
After PR #40444 when I try to update composer.lock I get:
```
phil@phil-Inspiron-5468:~/git/owncloud/core/apps/files_external/3rdparty$ composer update
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - files_external/3rdparty is present at version dev-master and cannot be modified by Composer
    - roave/security-advisories dev-latest conflicts with guzzlehttp/guzzle <6.5.8|>=7,<7.4.5 (files_external/3rdparty dev-master replaces guzzlehttp/guzzle *).
    - Root composer.json requires roave/security-advisories dev-latest -> satisfiable by roave/security-advisories[dev-latest].
```

And I have these versions of composer and PHP:
```
phil@phil-Inspiron-5468:~/git/owncloud/core/apps/files_external/3rdparty$ composer --version
Composer version 2.3.10 2022-07-13 15:48:23
phil@phil-Inspiron-5468:~/git/owncloud/core/apps/files_external/3rdparty$ php --version
PHP 7.4.32 (cli) (built: Sep 29 2022 22:24:52) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.32, Copyright (c), by Zend Technologies
    with Xdebug v3.1.5, Copyright (c) 2002-2022, by Derick Rethans
```

**Comment by phil-davis** [comment]
@DeepDiver1975 do you know what needs to be tweaked in `composer.json` to fix this?

**Comment by phil-davis** [comment]
Note: I want to bump phpseclib, which is used here and in the regular core composer.lock - so bumping both together would be nice. https://github.com/phpseclib/phpseclib/issues/1854

**Comment by DeepDiver1975** [comment]
roave/security-advisories dislikes our current used guzzle lib - so updating it should be enough.

Please note that if we have dupllicates in files_external and core we can resolve this the same way as done with files_primary_s3: https://github.com/owncloud/files_primary_s3/blob/80043ded3a89bdf608aaa7626dfd5eb356773f1d/composer.json#L8-L11

**Comment by DeepDiver1975** [comment]
@phil-davis https://github.com/owncloud/core/pull/40447

**Comment by phil-davis** [comment]
Fixed in PR #40449
